### PR TITLE
General CSS Fixes

### PR DESCRIPTION
**Pull request by jakeoid**
Was literally one pixel off which was driving me mad, accept PR, thanks.

### CODE REVIEW

File: theme.css [hunk 1]
```diff
@@ -273,7 +307,7 @@
 
 .settings .settings-header {
     background: #141414;
-}
```
Comment (TTtie) [inline theme.css]: 👏

Comment (jakeoid) [inline theme.css]: aaa my bad

**Comment by TTtie** [comment]
you miss off the ending bracket, add it kthx so you doj

**Comment by TTtie** [comment]
*don't bork it

**Comment by jakeoid** [comment]
Why not just pull request on my fork? :kappa:

**Comment by TTtie** [comment]
>mobile
k

**Comment by jakeoid** [comment]
What line number? Im having a Jakeoid

**Comment by jakeoid** [comment]
Nvm, got it.

**Pull request merged**